hackage-server.cabal

```diff
@@ -102,11 +102,8 @@ flag network-uri
   description: Get Network.URI from the network-uri package
   default: True
 
-executable hackage-server
-  if ! flag(build-hackage-server)
-    buildable: False
-  main-is: Main.hs
-  other-modules:
+library
```

Comment:
I'm actually fine with it; cabal 2.0 is about to be released anyway; the travis CI job runs on cabal 2.x; and there's very few people that actually run their own hackage-server instance so there's not many ppl that will be affected anyway, and I was about to set `cabal-version:>=2.0` in the `.cabal` file anyway :-)